app/main/views/auth.py

```diff
@@ -206,6 +228,11 @@ def confirm_dept():
 
     if form.validate_on_submit():
         if form.confirm.data == 'yes':
+
+            if current_app.config.get('DISABLE_INTERSTITIALS'):
```

Comment:
No, it just skips the interstitial pages, where the user is told they are being sent to login with their IDP or they are being sent back to the client service.

Checking for `app.config['TESTING']` would have been better in hindsight.

Yes, could be patched instead.